<HTML>
<HEAD>
<meta charset="UTF-8">
<title>RetryFlowException.<init> - application</title>
<link rel="stylesheet" href="../../../style.css">
</HEAD>
<BODY>
<a href="../../index.html">application</a>&nbsp;/&nbsp;<a href="../index.html">net.corda.v5.application.flows</a>&nbsp;/&nbsp;<a href="index.html">RetryFlowException</a>&nbsp;/&nbsp;<a href="./-init-.html">&lt;init&gt;</a><br/>
<br/>
<h1>&lt;init&gt;</h1>
<a name="net.corda.v5.application.flows.RetryFlowException$&lt;init&gt;(kotlin.String)"></a>
<code><span class="identifier">RetryFlowException</span><span class="symbol">(</span><span class="identifier" id="net.corda.v5.application.flows.RetryFlowException$<init>(kotlin.String)/message">message</span><span class="symbol">:</span>&nbsp;<a href="https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-string/index.html"><span class="identifier">String</span></a><span class="symbol">?</span><span class="symbol">)</span></code><br/>
<a name="net.corda.v5.application.flows.RetryFlowException$&lt;init&gt;(kotlin.Throwable)"></a>
<code><span class="identifier">RetryFlowException</span><span class="symbol">(</span><span class="identifier" id="net.corda.v5.application.flows.RetryFlowException$<init>(kotlin.Throwable)/cause">cause</span><span class="symbol">:</span>&nbsp;<a href="https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-throwable/index.html"><span class="identifier">Throwable</span></a><span class="symbol">?</span><span class="symbol">)</span></code><br/>
<a name="net.corda.v5.application.flows.RetryFlowException$&lt;init&gt;()"></a>
<code><span class="identifier">RetryFlowException</span><span class="symbol">(</span><span class="symbol">)</span></code><a name="net.corda.v5.application.flows.RetryFlowException$&lt;init&gt;(kotlin.String, kotlin.Throwable)"></a>
<code><span class="identifier">RetryFlowException</span><span class="symbol">(</span><span class="identifier" id="net.corda.v5.application.flows.RetryFlowException$<init>(kotlin.String, kotlin.Throwable)/message">message</span><span class="symbol">:</span>&nbsp;<a href="https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-string/index.html"><span class="identifier">String</span></a><span class="symbol">?</span><span class="symbol">, </span><span class="identifier" id="net.corda.v5.application.flows.RetryFlowException$<init>(kotlin.String, kotlin.Throwable)/cause">cause</span><span class="symbol">:</span>&nbsp;<a href="https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-throwable/index.html"><span class="identifier">Throwable</span></a><span class="symbol">?</span><span class="symbol">)</span></code>
<p>This exception will cause the flow to retry from its previous checkpoint.</p>
</BODY>
</HTML>
